EIPS/eip-721.md

```diff
@@ -444,4 +444,4 @@ XXXXERC721, by William Entriken -- a scalable example implementation
 
 ## Copyright
 
-Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```

Comment:
This didn't change anything. Please don't make me redo this entire PR because of this mistake.

Reply:
You can just remove this file/diff from the PR.

Reply:
Done